src/GraphQL/Transports/Json/WebSocketMessage.cs

```diff
@@ -0,0 +1,24 @@
+namespace GraphQL.Transports.Json
+{
+    public class WebSocketMessage
+    {
+        public const string ID_KEY = "id";
+        public const string TYPE_KEY = "type";
+        public const string PAYLOAD_KEY = "payload";
+
+        /// <summary>
+        /// Nullable Id
+        /// </summary>
+        public string? Id { get; set; }
+
+        /// <summary>
+        /// Type of packet
+        /// </summary>
+        public string? Type { get; set; }
+
+        /// <summary>
+        /// Nullable payload
+        /// </summary>
+        public object? Payload { get; set; }
+    }
```

Comment:
I guess I don't usually see `ToString` implemented on data classes.  I guess we could, but it would only represent a partial view of the data since the payload type is unknown and could be complex.  Perhaps a debugger display attribute would be better if it's necessary.

https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/enhancing-debugging-with-the-debugger-display-attributes

Reply:
I did the same in parser - removed unused `ToString()` overrides. Let's keep as is, without debuggerdisplay attribute.